src/popup/pages/Popups/MessageSign.vue

```diff
@@ -90,12 +89,13 @@ export default defineComponent({
     }));
 
     function cancel() {
-      props.reject(new Error('Rejected by user'));
+      popupProps.value?.reject(new Error('Rejected by user'));
```

Comment:
Please use `RejectedByUserError` instead. Probably you didn't rebased your branch for a while - on current develop this was already changed.